src/multiplexer/EventHandler.cpp

```diff
@@ -152,13 +165,14 @@ void EventHandler::handleResponse(int clientFd) {
 	// replace the below
 	std::cout << "Sending response to client " << clientFd << "\n";
 	_clients[clientFd]->resetData();
-	const char* response =
-		"HTTP/1.1 200 OK\r\n"
-		"Content-Type: text/plain\r\n"
-		"Content-Length: 13\r\n"
-		"\r\n"
-		"Hello, World!";
-	write(clientFd, response, strlen(response));
+	//const char* response =
+	//	"HTTP/1.1 200 OK\r\n"
+	//	"Content-Type: text/plain\r\n"
+	//	"Content-Length: 13\r\n"
+	//	"\r\n"
+	//	"Hello, World!";
+	//write(clientFd, response, strlen(response));
+	cgiOut(clientFd, "cgi_bin/tester.cgi");
```

Comment:
just a heads up, the actual executable that will be used for the CGI's are going to be in the config file, specificed by the user

lgtm👍